searchcore/src/vespa/searchcore/proton/server/searchabledocsubdb.cpp

```diff
@@ -91,19 +93,19 @@ createIndexManagerInitializer(const DocumentDBConfig &configSnapshot, SerialNum
                               const IndexConfig &indexCfg,
                               std::shared_ptr<searchcorespi::IIndexManager::SP> indexManager) const
 {
-    Schema::SP schema(configSnapshot.getSchemaSP());
+    const Schema::SP & schema(configSnapshot.getSchemaSP());
```

Comment:
please decide if you want a shared pointer (use value) or if you just want to inspect the inner state of an object (use const reference to actual object directly).

Also, the shared pointer itself is not an inherently shareable object.

Reply:
Only reference was needed. Fixed.